CHANGELOG.md

```diff
@@ -1,48 +1,52 @@
-# 5.9.0
+# Upcoming
+
+* BREAKING CHANGE: Seperate the code from the angular_token_auth into seperate files.
+
+## 5.9.0
 
 * Upgrade Angular compatibility to include 1.3, 1.4, and 1.5
 
-# 5.8.0
+## 5.8.0
```

Comment:
Since this is now a PR into a feature branch, it might have been better to keep these changelog number updates in another PR, but don't worry about it now.